### PR TITLE
MAGECLOUD-3678: Add validation for node

### PR DESCRIPTION
**Pull request by shiftedreality**
### Fixed Issues (if relevant)

1. https://magento2.atlassian.net/browse/MAGECLOUD-3678

### Manual testing scenarios

1. https://jira.corp.magento.com/browse/MAGECLOUD-123

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: dist/docker/bin/docker [hunk 1]
```diff
@@ -6,16 +6,16 @@ if [ ${#@} -ne 0 ] && [ "${@#"-h"}" = "" ]; then
...
+  pull              pull latest images
+  up                destroy, re-create and start containers
+  down              destroy containers
+  bash              connect to bash
+  stop              stop containers
+  start             start containers
+  restart           restart containers
+  ece-build         build application
+  ece-deploy        deploy application
+  ece-redeploy      re-build and re-deploy application
```
Comment (NadiyaS) [inline dist/docker/bin/docker:18]: We do not follow BC in command naming and we are ok with renaming them without leaving aliases for old ones?

Comment (shiftedreality) [reply dist/docker/bin/docker:18]: It was not released yet. Only in `develop` branch

Comment (NadiyaS) [reply dist/docker/bin/docker:18]: I see

**Review by NadiyaS — APPROVED** [review]

**Review by billygilbert — APPROVED** [review]

**Comment by andriyShevtsov** [comment]
QA approved

**Pull request merged**